package/yast2-sap-ha.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Mar 15 17:38:52 UTC 2024 - Peter Varkoly <varkoly@suse.com>
+
+- yast2-sap-ha: Error occurred during the unattended installation: undefined class/module SapHA::Configuration::ClusterFinalizer
```

Comment:
I would split this line to two. I am not sure if it will be readable in `rpm -q --changelog`